single-page.html

```diff
@@ -1467,7 +1464,10 @@ <h2 class="no-num no-toc no-ref heading settled" id="status"><span class="conten
     technical report can be found in the <a href="https://www.w3.org/TR/"><abbr title="World Wide Web Consortium">W3C</abbr> technical
     reports index</a> at https://www.w3.org/TR/.</em> </p>
 
-<p><a href="https://github.com/w3c/html/labels/errata">Errata</a> for this document are recorded as Github issues with the "errata" label.The <a href="https://w3c.github.io/html/">latest HTML editors' draft</a> shows proposed resolutions of errata <i>in situ</i>.</p>
+
+   <p>This specification defines the 2nd edition of the 5th major version, first minor revision of the core language of the World Wide Web: the Hypertext Markup Language (HTML). In this version, a significant error with <a href="https://w3.org/tr/51/">HTML 5.1</a> has been rectified, as noted in the <a href="errata">Errata corrected</a> section.</p>
+
+<p><a href="https://github.com/w3c/html/labels/errata">Errata</a> for this document are recorded as Github issues with the "errata" label. The <a href="https://w3c.github.io/html/">latest HTML editors' draft</a> shows proposed resolutions of errata <i>in situ</i>.</p>
```

Comment:
It does seem weird to mix issues for the current editor's draft (or future draft) for which we intend to drive to zero, with errata issues that persist indefinitely for past branches of the spec. I think it might be better to just maintain a separate *.md document (separately) for errata--so that it doesn't show up in the issues list.